src/components/organize/tasks/forms/TaskDetailsForm.tsx

```diff
@@ -1,13 +1,14 @@
 import { Form } from 'react-final-form';
 import { useQuery } from 'react-query';
 import { useRouter } from 'next/router';
+import validator from 'validator';
```

Comment:
Can you say something about why you chose the `validatator` package? Are there any other packages that you have looked at and what are the pros and cons?

Reply:
To be honest, I didn't compare it thoroughly with any other package. I found a few packages but was immediately convinced to use this one. Because:
- It's light.
- It has many more features for string sanitization and validation. (e.g: `isURL()`, `isSlug()`, `isEmail()`, `isPostalCode()` etc.)
- it appears to have a very strong community behind it. Not just one single developer.
- its weekly downloads reach ~6M times while it has been around for a very long time.

Of the reasons above, the second was the most convincing for me, for I know we might end up using some of its other features.